src/pages/organizers/[organizerId]/preview/index.page.tsx

```diff
@@ -0,0 +1,261 @@
+import getConfig from 'next/config';
+import { useRouter } from 'next/router';
+import { useMemo, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { dehydrate, useQueryClient } from 'react-query';
+
+import {
+  useGetOrganizerByIdQuery,
+  useGetOrganizerPermissions,
+} from '@/hooks/api/organizers';
+import {
+  OwnershipRequest,
+  RequestState,
+  useGetOwnershipRequestsQuery,
+  useRequestOwnershipMutation,
+} from '@/hooks/api/ownerships';
+import { useGetUserQuery } from '@/hooks/api/user';
+import { SupportedLanguage } from '@/i18n/index';
+import { Organizer } from '@/types/Organizer';
+import { Alert, AlertVariants } from '@/ui/Alert';
+import { Box } from '@/ui/Box';
+import { Button, ButtonVariants } from '@/ui/Button';
+import { Icons } from '@/ui/Icon';
+import { Inline } from '@/ui/Inline';
+import { Modal, ModalSizes, ModalVariants } from '@/ui/Modal';
+import { Page } from '@/ui/Page';
+import { Stack } from '@/ui/Stack';
+import { getApplicationServerSideProps } from '@/utils/getApplicationServerSideProps';
+import { getLanguageObjectOrFallback } from '@/utils/getLanguageObjectOrFallback';
+
+import { OrganizerTable } from './OrganizerTable';
+
+const OrganizersPreview = () => {
+  const { t, i18n } = useTranslation();
+  const queryClient = useQueryClient();
+  const [isQuestionModalVisible, setIsQuestionModalVisible] = useState(false);
+  const [isSuccessAlertVisible, setIsSuccessAlertVisible] = useState(false);
+  const [isErrorAlertVisible, setIsErrorAlertVisible] = useState(false);
+  const router = useRouter();
+  const { publicRuntimeConfig } = getConfig();
+  const isOwnershipEnabled = publicRuntimeConfig.ownershipEnabled === 'true';
+
+  const organizerId = useMemo(
+    () => router.query.organizerId as string,
+    [router.query.organizerId],
+  );
+  const getOrganizerByIdQuery = useGetOrganizerByIdQuery({
+    id: organizerId,
+  });
+
+  const getOrganizerPermissionsQuery = useGetOrganizerPermissions({
+    organizerId: organizerId,
+  });
+
+  // @ts-expect-error
```

Comment:
Can you add explicit type casts for the queries? I know it is not as clean, but I prefer we have some type safety instead of just ignoring ts.